python/ngen_cal/src/ngen/cal/parameter.py

```diff
@@ -11,5 +11,13 @@ class Parameter(BaseModel, allow_population_by_field_name = True):
     min: float
     max: float
     init: float
+    alias: Optional[str]
```

Comment:
Lets change this to the following since `alias` is non-`None`, we just rely on the validator to set `alias` to `name` if it was not provided. Just will avoid having to check if `alias` is `None` else where in the code to please static analysis tools.

```suggestion
    alias: str = None
    """Set to value of `name` if `alias` is not provided."""
```